src/components/Welcome/index.tsx

```diff
@@ -70,14 +69,9 @@ const WelcomeActions: FunctionComponent<WelcomeActionsProps> = ({
 }
 
 const Welcome: FunctionComponent<StepComponentBaseProps> = ({ nextStep }) => {
-  const { steps } = useSdkOptions()
+  const [, { findStep }] = useSdkOptions()
```

Comment:
Why is this now written this way instead of `[{ findStep }]` or `[undefined, { findStep }]`?

Reply:
As I mentioned above:
> Refactor `useSdkOptions()` hook to return a tuple: `[options, enhancedOptions]`

so `const [, { findStep }] = useSdkOptions()` just a shorthand of `const { findStep } = useSdkOptions()[1]`. Because we don't need `options` here, so we don't want to declare it

Reply:
Ah, ok. Makes sense now. Thanks for the explanation.